examples/ili9341_shield_simpletest.py

```diff
@@ -14,21 +16,53 @@
 
 displayio.release_displays()
 display_bus = displayio.FourWire(spi, command=tft_dc, chip_select=tft_cs)
-
 display = adafruit_ili9341.ILI9341(display_bus, width=320, height=240)
 
 # Make the display context
 splash = displayio.Group(max_size=10)
 display.show(splash)
 
+# Draw a green background
 color_bitmap = displayio.Bitmap(320, 240, 1)
 color_palette = displayio.Palette(1)
-color_palette[0] = 0xFF0000
+color_palette[0] = 0x00FF00 # Bright Green
 
 bg_sprite = displayio.TileGrid(color_bitmap,
                                pixel_shader=color_palette,
                                x=0, y=0)
+
 splash.append(bg_sprite)
 
+# Draw a smaller inner rectangle
+inner_bitmap = displayio.Bitmap(280, 200, 1)
+inner_palette = displayio.Palette(1)
+inner_palette[0] = 0x65328F # Blinka Purple
+inner_sprite = displayio.TileGrid(inner_bitmap,
+                                  pixel_shader=inner_palette,
+                                  x=20, y=20)
+splash.append(inner_sprite)
+
+# Draw some text the manual way!
```

Comment:
i think we should use label, cause this is kinda intense

Reply:
Ok. I tried to make it as basic as possible, but I agree that it adds a bit to it.

Reply:
yah i think it got too basic :D